### ISSUE TITLE
recording

### ISSUE THREAD

**Issue by ayskura** [steffest/BassoonTracker#74]
it would be awesome to be able to record directly sounds as in Fast Tracker 2 etc, I believe that's really most of the magic of trackers relies on being able to record and play very quickly

**Comment by steffest** [comment]
Indeed - in fact it would be quite trivial to add, but I never could see a good use case.
Back in the FT2 days - when you only could run 1 DOS program at a time - it made sense to also integrate the recording, but these days, it would make more sense to just use a separate dedicated app for that.

**Comment by ayskura** [comment]
Yes of course today is much easier to sample, but the difference of sampling nd playing directly that sample is what in my opinion makes it so nice to use, for speed and ease of sampling.
On android I can chose to open a recorder app, so it’s quite seamless actually, but on iPhone and computer not asks for using only the camera :(

Also I tried to host it on a website, just copying the whole thing but I couldn’t get it to work… I really love it, it would be awesome to have it as an APK / app standalone too as it’s one of the best trackers around, my compliments

Ays
